app/javascript/components/sidebar/area-of-interest/component.js

```diff
@@ -1,21 +1,44 @@
-import React, { PureComponent, Fragment } from 'react';
+import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
+import isEmpty from 'lodash/isEmpty';
 
 // styles
 // import './styles.scss';
 
 class AreaOfInterestComponent extends PureComponent {
-  static propTypes = {}
+  static propTypes = {
+    area: PropTypes.object.isRequired,
+    drawing: PropTypes.bool.isRequired,
+    setClearing: PropTypes.func.isRequired,
+    setAreaOfInterest: PropTypes.func.isRequired
+  }
 
-  static defaultProps = {}
+  onClear = () => {
+    this.props.setClearing(true);
+  }
+
+  toggleDrawing = (value) => {
+    this.props.setAreaOfInterest({ ...this.props, drawing: !value });
+  }
 
   render() {
-    const {} = this.props;
+    const { drawing, area } = this.props;
 
     return (
       <div className="c-area-of-interest">
-        Area of Interest
+        <button
+          onClick={() => this.toggleDrawing(drawing)}
+        >
+          { drawing ? 'Cancel' : 'Create Area' }
+        </button>
+
+        {!isEmpty(area) &&
+          <button
+            onClick={() => this.onClear()}
```

Comment:
You don't need to declare an arrow function in these cases. Only if you need to pass a param. Like the `toggleDrawing` function